pkg/verifier/notation/certStoresByType.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright The Ratify Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package notation
+
+import (
+	"context"
+
+	"github.com/deislabs/ratify/internal/logger"
+	"github.com/notaryproject/notation-go/verifier/truststore"
+)
+
+type verificationCertStores map[string]interface{}
+
+// type certStores map[string][]string
+type certStoresByType map[string]map[string][]string
```

Comment:
We can define a certStores interface with `GetCertGroup` API, `certStoresByType` just implements this interface. Then we could mock it in the `truststore_test`.